src/settings.py

```diff
@@ -261,8 +261,9 @@ class GlobalSettings:
     llm_openai_max_response_tokens = 250
 
     # parameters for open loop LLM approach
-    open_loop_llm_model_name = "text-curie-001"
-    open_loop_llm_temperature = 0.0
+    open_loop_llm_model_name = "text-curie-001"  # "text-davinci-002"
+    open_loop_llm_temperature = 0.5
+    open_loop_llm_num_completions = 1
```

Comment:
@Alchemy5 take note that I'm using small defaults here. we will override them in your script with the command line for the experiments